qiskit/providers/aer/jobs/aerjobset.py

```diff
@@ -37,27 +37,25 @@
 
 
 class AerJobSet(Job):
-    """A set of cluster jobs.
+    """A set of :class:`AerJob` class for Qiskit Aer simulators.
 
     An instance of this class is returned when you submit experiments with
-    executor option.
-    It provides methods that allow you to interact
+    executor option.  It provides methods that allow you to interact
     with the jobs as a single entity. For example, you can retrieve the results
     for all of the jobs using :meth:`result()` and cancel all jobs using
     :meth:`cancel()`.
     """
 
-    def __init__(self, backend, job_id, func, experiments: List[QasmQobj], executor=None):
+    def __init__(self, backend, job_id, fn, experiments: List[QasmQobj], executor=None):
         """AerJobSet constructor.
 
         Args:
             backend(Aerbackend): Aerbackend.
             job_id(int): Job Id.
-            func(fun): Callabled function.
+            fn(function): Callabled function.
```

Comment:
```suggestion
            fn(function): Callable function.
```

We should probably describe what this function is for, and what inputs/outputs it should expect.

Reply:
I intend to add the following comments. What do you think? 
fn(function): a callabled function to execute qobj on backend. :class:`AerBackend` sets :meth:`AerBackend._run()` function that takes qobj(QasmQobj) and job_id(string) as an argument.